site/releases/2024-oct-22/release-notes.qmd

```diff
@@ -0,0 +1,513 @@
+  <iframe src="/training/training.html" width="540" height="325" style="border:none; border-radius: 5px;"></iframe>
+  <a href="/training/training.html" target="_blank" style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; z-index: 10;"></a>
+</div>
+:::
+
+::::
+
+<!---
+PR #446: Embed Validating Models 101 playlist into Guides section
+URL: https://github.com/validmind/documentation/pull/446
+Labels: documentation
+--->
+### Validating models 101 videos
+
+We created a series of short videos to help you better understand the model validation process. These videos introduce you to the steps you need to follow when validating models on our platform. 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?si=JfBNClmsZYLS7m9F&amp;list=PL7I7ZrCoVCCObEiuLLAL0Wm9GQ6x5U0q5" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen style="box-shadow: 5px 5px 5px #ccc, -5px 5px 5px #ccc; border-radius: 5px;"></iframe>
+
+<!---
+PR #450: Embed "How do I ... add tests?" video in developer docs
+URL: https://github.com/validmind/documentation/pull/450
+Labels: documentation
+--->
+### Adding tests video
+
+We added a short FAQ-style video to show you how to find the tests ValidMind provides and add them to your own model documentation. 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/5J7wysDIXgI?si=KPkFhx3O6NknijRf" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen style="box-shadow: 5px 5px 5px #ccc, -5px 5px 5px #ccc; border-radius: 5px;"></iframe>
+
+<!---
+PR #459: Deployment architecture docs
+URL: https://github.com/validmind/documentation/pull/459
+Labels: documentation
+--->
+### New deployment options overview
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-70-ns}
+We expanded our documentation on deployment options for ValidMind. You can now find detailed information on both multi-tenant cloud and single-tenant options, giving you more clarity on how to deploy ValidMind based on your needs.
+:::
+
+::: {.w-30-ns .tc}
+[Learn about deployment options {{< fa book-open >}}](/about/deployment/deployment-options.qmd){.button}
+:::
+::::
+
+<!---
+PR #460: Add link to GitHub for code samples
+URL: https://github.com/validmind/documentation/pull/460
+Labels: documentation
+--->
+### Find our open-source on GitHub
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-70-ns}
+To make it easier to find our open-source software on GitHub, we added a link to the code samples page. 
+:::
+
+::: {.w-30-ns .tc}
+[{{< fa brands github >}} Try it on GitHub](https://github.com/validmind/developer-framework){.button}
+:::
+
+::::
```

Comment:
TO DO: Our instructions say to add `{{< include /releases/_how-to-upgrade.qmd >}}` at the end, but the file seems to be missing from our repo.